DafnyRuntimeGo/Makefile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I don't think we need this file. This project is already in Go, and let's use the folder name from Dafny with `-gomod` suffix

Reply:
We are going to delete this directory in some point. So, I will leave this as is it.